### PR TITLE
Exotic Rentals V.1.1.0

### PR DESCRIPTION
**Pull request by NickEmma**
# Hi there 👋🏻

**Thanks for taking the time to review the project. Here are some things important to note about it:**

### Frontend Link

You can find the link for the frontend project [here](https://github.com/NickEmma/car-rental-app/pull/10)

## Now into the project...

**This project was done following git-flow, for which we have different pull requests, each of them with specific information about each feature. You can refer to them below**

- ✅ https://github.com/NickEmma/car-rental-api/pull/18. Here was the basic project configuration.
- ✅ https://github.com/NickEmma/car-rental-api/pull/19. Here we add the JSON format using an active record serializer.
- ✅https://github.com/NickEmma/car-rental-api/pull/20. We updated the routes and fix bugs.
- ✅https://github.com/NickEmma/car-rental-api/pull/21. We deployed the backend using the railway. app and test the endpoints.
- ✅ https://github.com/NickEmma/car-rental-api/pull/22. We tested the functionalities and use swagger to test the routes.
- ✅ https://github.com/NickEmma/car-rental-api/pull/23. We loaded the database with random data.
- ✅ The project follows the style guidelines 
- ✅ The project was developed following the TDD approach. Thus, almost every feature is tested

# Final notes 📔

_Is expected that this project meets the requirements. Please let me know if anything is left to be done or can be improved. Again, thanks for the review._

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,194 @@
...
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
```
Comment (Meltrust) [inline README.md:105]: - Kindly include the `rails db:create, rails db:migrate, rails db:seed` so your users are able to run your project. 👍

**Review by Meltrust — CHANGES_REQUESTED** [review]
_Note: Please, go to your frontend PR to see my comments on that APP._

Hi @NickEmma,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!






### To highlight:

-  Tests are passing✔️
-  App is is working well✔️ 
-  Good readme ✔️ 




#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._

_I strongly recommend you take them into account as they can improve your final evaluation._





Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

File: README.md (hunk 1 above)
Comment (NickEmma) [reply README.md]: @Meltrust Thank you for your review

Comment (Said-laasri) [reply README.md]: @Meltrust thanks for the review . I made the required changes . :+1:

**Comment by mcrd25** [comment]
Hi Team, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Review by mcrd25 — APPROVED** [review]
Hi Team, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by Said-laasri** [comment]
@mcrd25  thanks you for reviewing our project.

**Pull request merged**